pincer/_config.py

```diff
@@ -10,21 +10,21 @@ class GatewayConfig:
     """This file is to make maintaining the library and its gateway
     configuration easier.
     """
-    socket_base_url: str = "wss://gateway.discord.gg/"
+    MAX_RETRIES: int = 5
     version: int = 9
     encoding: str = "json"
     compression: Optional[str] = "zlib-stream"
```

Comment:
```suggestion
    compression: str = "zlib-stream"
```

Reply:
This should be `Optional` because you can set it to `None` to disable compression if you want to do that for some reason.

Reply:
not sure if Optional fits either way in this case

Reply:
Yeah, I changed it to be just `str` since you can leave it blank for no compression.